src/modules/endpoint/MemoryDatabase.js

```diff
@@ -184,6 +206,72 @@ class MemoryDatabase {
     }))
   }
 
+  writeEvents (): Promise<number> {
```

Comment:
why does writeEvents return a Promise, but readEvents is async? Can we make writeEvents an async function?

Reply:
I fixed this in #35, you don't necessarily have to fix it here.